snap/squashfs/squashfs.go

```diff
@@ -117,6 +118,92 @@ func (s *Snap) ReadFile(filePath string) (content []byte, err error) {
 	return ioutil.ReadFile(filepath.Join(unpackDir, filePath))
 }
 
+// skippper could also be a map[string]bool, but _as it's only supposed to be
```

Comment:
You write what it could be - but what is it ;) ? Maybe something like `skipper helps keeping track of already seen paths` or something? And then the rest of the comment.